pypeit/sensfunc.py

```diff
@@ -462,126 +693,273 @@ def write_QA(self):
 # TODO Add a method which optionally merges sensfunc using the nsens > 1 logic
 
 
+class IRSensFunc(SensFunc):
+    """
+    Determine a sensitivity functions from standard-star spectra. Should only
+    be used with NIR spectra (:math:`\lambda > 7000` angstrom).
```

Comment:
I think we are heading to a mode where we use this for all spectrographs and wavelengths.